libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/map/marker/MapMarkerFactory.kt

```diff
@@ -10,9 +10,6 @@ import com.mapbox.maps.extension.style.layers.properties.generated.IconAnchor
 import com.mapbox.maps.plugin.annotation.generated.PointAnnotationOptions
 import com.mapbox.navigation.dropin.util.BitmapMemoryCache
 
-/**
- * Factory class for creating all drop-in UI map point annotations.
- */
```

Comment:
@abhishek1508 Why did you remove this comment?

Reply:
In the beginning all our classes were public and hence the need for documentation for all. Since we changed the relevant ones to internal, we did not remove the comments. Our code base has not become quite inconsistent as to some `internal` classes have descriptions vs the others don't. This was an attempt to bring consistency. 

However I have put them back for the time being.